README.md

```diff
@@ -34,3 +43,21 @@ The server will store some statistics data every 12 hours for analysing the traf
 - The number of push notifications sent to Android devices
 
 There is also an endpoint `/get_statistics_data` to get the data above.
+
+
+## Potential issues
+
+If you get an issue during the `sudo apt update` about the certificate chain being invalid,
+try to create a file at `/etc/apt/apt.conf.d/99deboxenio-cert` with content:
+`Acquire::https::deb.oxen.io::Verify-Peer "false";`
+
+
+If you get issue with the Rust compiler, use
+`pip3 install setuptools_rust docker-compose` (or just `pip3 install setuptools_rust`)
```

Comment:
Where do we have rust in here?

Reply:
looks like my issue was related to the pip inside my virtualenv being too old. 

`pip install --upgrade pip` and removing that setuptool_rust is fixed doing this.
For references, here is a log file on what I was getting as error
[Uploading logs_rust.log…]()